update.rdf

```diff
@@ -1,30 +0,0 @@
-<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Please don't remove this file. It is required for a smooth update from the current Zotero 6 plugin to a new one which supports both Zotero 6 and Zotero 7.